ROLES.md

```diff
@@ -164,6 +165,47 @@ a 180 day period, than that individual may lose membership. On-going contributio
 
 Members who frequently contribute code are expected to proactively perform code reviews for the area that they are active in.
 
+## Documentation Reviewer
+
+A **Documentation Reviewer** is trusted to only approve
+content that meets the acceptance criteria described in the
+[contribution guides](https://istio.io/about/contribute/).
+
+### Requirements
+
+To become a **Reviewer**, contributors must meet the following **requirements**:
+
+* Be a **Member** of the Istio community.
+* Perform 5 substantial contributions to the `istio.io` repo. Substantial
+  contributions include the following examples:
+
+    * New content
+    * Content reviews
+    * Content improvements
+
+* Demonstrate a solid commitment to documentation quality and use of our style
+  guide.
+* Be sponsored by a Istio Maintainer or WG Lead.
+
+### Responsibilities
+
+A **Reviewer** would have the following **responsibilities** in Istio:
+
+* Review PRs in `istio/istio.io` following the process described in the
+  [How to review content](https://istio.io/about/contribute/github/#review) guide.
+* Ensure the relevant technical Working Group is added as a reviewer and ensure
+  a WG lead or maintainer has approved the PR.
+
+### Privileges
+
+A **Reviewer** would have the following **privileges**:
+
+* Content approved by a **Reviewer** gets prioritized by Maintainers and WG Leads.
+* Reviewers can place a `pre-reviewed` label to notify the Docs WG Leads and
+  Maintainers to expedite publication of the pre-reviewed content.
+
+Only Docs Maintainers and WG Leads can merge content into the `istio/istio.io` master.  
```

Comment:
Maybe this should be in a note to indicate that this is explicitly a privilege that a doc reviewer does not have?

Reply:
Added text making the limitation more explicit.